### ISSUE TITLE
Drumset voices not playing after the Nth one

### ISSUE THREAD

**Issue by jpcima** [sfztools/sfizz#243]
This happens with the ARIA mapping of the Naked Drums set.

The following happens: when you play a note 4 times in moderately fast succession, the voice plays, but the next ones don't. One has to wait some seconds until the key is playable again.

This may have to do with a particularity of this set: all samples are 2 seconds long wave files, ending with a silence period. Often, the silence part occupies half the sound length or more.

**Comment by paulfd** [comment]
Do you happen to have the sfz file? Apparently the kontakt version is free nowadays.

**Comment by redtide** [comment]
[here](https://www.dropbox.com/s/8w5apu1ty87lpku/Naked_Drums_aria_sfz.rar).

**Comment by paulfd** [comment]
This is one hell of an include pattern :smile:

**Comment by paulfd** [comment]
Using the sfz flattener you see the polyphony and off_time uses clearly.
```
...
<master> amplitude_oncc42=200 master_label=Mixture of Sabian/Zildjian - Ride polyphony=4 off_time=1.6 master_volume=-3 
<control> default_path=../Samples/Naked_Drums_Metal_Cymbals Samples/ 
...
```
This should be handled somewhat OK by #275 , while `off_time` will require overflow voices and probably a further PR. I'm tagging the issue for the release after next.

**Comment by paulfd** [comment]
This will be solved (if not already) by #275
